### PR TITLE
Feature/cp matrix

### PR DESCRIPTION
**Pull request by Bluesmile82**
Add cp matrix table with tabs and legend

image

### CODE REVIEW

File: app/javascript/components/tpi/charts/cp-matrix/cp-matrix-table.jsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
+/* eslint-disable function-paren-newline */
+/* eslint-disable implicit-arrow-linebreak */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Table from 'rc-table';
+import ReactTooltip from 'react-tooltip';
+import cx from 'classnames';
+import chevronIcon from 'images/icon_chevron_dark/chevron_down_black-1.svg';
+
+const COLORS = {
```
Comment (martintomas) [inline app/javascript/components/tpi/charts/cp-matrix/cp-matrix-table.jsx:10]: List of available names can be seen at https://github.com/Vizzuality/laws_and_pathways/blob/develop/app/models/cp/alignment.rb#L27. Based on this list, `Not aligned` should be `Not Aligned`. I will also add `Not assessable using TPI’s methodology` on BE side to this list.

**Review by martintomas — CHANGES_REQUESTED** [review]
Looks good to me, just two minor points:
 - matrix is not responsive, it is not issue for matrix itself, but tabs and legend should change appropriately. Maybe it is not part of this PR
 - I see kind of artefact at right border of matrix (probably for scrollbar). I would be great to get rid of it, but no big deal
Screenshot 2023-07-06 at 9 37 25

File: app/javascript/components/tpi/charts/cp-matrix/cp-matrix-table.jsx (hunk 1 above)
Comment (martintomas) [reply app/javascript/components/tpi/charts/cp-matrix/cp-matrix-table.jsx:10]: I have covered this with my commit ;)

**Review by martintomas — APPROVED** [review]

**Comment by Bluesmile82** [comment]
Hey! Thanks for the review! I have updated it to be responsive. I can't reproduce the artifact in Chrome, Firefox or Safari though

**Pull request merged**